libs/template/jest.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   bail: true,
   preset: '../../jest.preset.js',
   coverageReporters: ['lcov', 'cobertura'],
-  coverageDirectory: '../../docs/test-coverage/template',
+  coverageDirectory: '../../dist/apps/docs/test-coverage/template',
```

Comment:
The build output directory for the `docs` project is now `dist/apps/docs` instead of the raw files in `docs`.